src/aura/HH_AddressMgr/HH_AddressMgr.cmp

```diff
@@ -0,0 +1,159 @@
+                                Select Existing Address
+                            </h4>
+                        </div>
+                        <div class="slds-section__content">                                                    
+                            <ul class="slds-list--vertical slds-has-cards--space slds-has-list-interactions" >
+                                <aura:iteration items="{!v.listAddr}" var="addr" indexVar="iAddr" >
+                                    <li class="{!iAddr == v.iAddrSelected ? 'slds-list__item selected' : 'slds-list__item' }" 
+                                        data-iAddr="{!iAddr}" onclick="{!c.onPressAddrTile}" >   
+                                        <div class="slds-tile slds-tile--board">
+                                            <div class="slds-tile__detail" >
+                                                <!-- address -->
+                                                <div class="slds-text-body--small" >
+                                                    <ui:outputText value="{!addr.MailingStreet__c + 
+                                                    	(addr.MailingStreet2__c != null ? '\r' + addr.MailingStreet2__c : '')}" /><br/>
+                                                    <ui:outputText value="{!addr.MailingCity__c + (addr.MailingState__c != null ? ', ' : '') + addr.MailingState__c + ' ' + addr.MailingPostalCode__c + ' '}" /><br/>
+                                                    <ui:outputText value="{!addr.MailingCountry__c}" />
+                                                </div>
+                                            </div> 
+                                        </div>
+                                    </li>
+                                </aura:iteration>
+                            </ul>
+                        </div>
+                    </div>
+
+                    <!-- New Address Section -->
+                    <div class="{!v.isExistingAddrSectionOpen ? 'slds-section' : 'slds-section slds-is-open'}">
+                        <div class="slds-section-title--divider slds-p-around--x-small">
+                            <h4 class="slds-section__title-action" onclick="{!c.toggleAddrSection}">
+                                <c:svg class="slds-button__icon"
+                                       xlinkHref="/resource/SLDS/1_0_3/assets/icons/utility-sprite/svg/symbols.svg#switch" />
+                                Enter a New Address
+                            </h4>
+                        </div>
+                        <div class="slds-section__content">                                                    
+                            <div class="slds-form--stacked">
+                                <div class="slds-form-element">
+                                    <label class="slds-form-element__label" for="inputSample2">Mailing Street</label>
```

Comment:
You may want to remove the `for` attribute of the labels in this section, since they all refer to the same non-existent element.  Alternatively, add unique ids to the input elements and use those ids for these attributes.  This is non-breaking, so probably optional to change.